src/components/auth/authContent.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {useState} from 'react';
+import {Text, View, StyleSheet} from 'react-native';
+import {InputField} from '../../components';
+import {ButtonInput} from '../../components/input/button';
+import {AccountStore} from '../../store/account/accountStore';
+import {useNavigation} from '@react-navigation/native';
+interface AuthContentProps {
+  onSuccessLogin: () => void;
+}
+
+export const AuthContent = ({onSuccessLogin}: AuthContentProps) => {
+  const {setUser} = AccountStore();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [passwordError, setPasswordError] = useState(false);
+
+  const setUserEmail = (item: string) => {
+    setEmail(item);
+  };
+
+  const handlerButton = () => {
+    try {
+      if (password === 'a' && email === 'a') {
```

Comment:
May be better use storage to store this data.
Try look for md5 to obfuscate password and save in AsyncStorage.